homework/src/index.js

```diff
@@ -32,12 +32,108 @@
 
```

Comment:
In general remark upfront: there are a lot of ESLint warnings for your code, in addition to some spelling errors. You will probably have seen them in VSCode as wavy underlines. You should not leave such warnings uncorrected. Strive to have zero of these warnings. Here is the list:

```

/Users/jimcramer/hackyourfuture/class16/satambanja/homework/src/index.js
   37:95  warning  Missing semicolon                            semi
   40:55  warning  Missing semicolon                            semi
   41:65  warning  Missing semicolon                            semi
   42:66  warning  Missing semicolon                            semi
   49:17  warning  'option' is assigned a value but never used  no-unused-vars
   57:1   warning  More than 2 blank lines not allowed          no-multiple-empty-lines
   62:32  warning  Missing semicolon                            semi
   63:15  warning  Missing semicolon                            semi
   64:1   warning  More than 2 blank lines not allowed          no-multiple-empty-lines
   69:13  warning  Missing semicolon                            semi
   71:11  warning  Missing semicolon                            semi
   72:1   warning  More than 2 blank lines not allowed          no-multiple-empty-lines
   79:29  warning  Missing semicolon                            semi
   81:47  warning  Missing semicolon                            semi
   82:1   warning  More than 2 blank lines not allowed          no-multiple-empty-lines
   85:11  warning  Missing semicolon                            semi
   86:1   warning  More than 2 blank lines not allowed          no-multiple-empty-lines
   94:52  warning  Missing semicolon                            semi
   95:48  warning  Missing semicolon                            semi
   96:15  warning  'td1_1' is assigned a value but never used   no-unused-vars
   96:15  warning  Identifier 'td1_1' is not in camel case      camelcase
   96:93  warning  Missing semicolon                            semi
   97:15  warning  Identifier 'td1_2' is not in camel case      camelcase
   97:15  warning  'td1_2' is assigned a value but never used   no-unused-vars
   97:71  warning  Missing semicolon                            semi
  100:49  warning  Missing semicolon                            semi
  101:15  warning  Identifier 'td2_1' is not in camel case      camelcase
  101:15  warning  'td2_1' is assigned a value but never used   no-unused-vars
  101:94  warning  Missing semicolon                            semi
  102:15  warning  Identifier 'td2_2' is not in camel case      camelcase
  102:15  warning  'td2_2' is assigned a value but never used   no-unused-vars
  102:79  warning  Missing semicolon                            semi
  104:49  warning  Missing semicolon                            semi
  105:15  warning  'td3_1' is assigned a value but never used   no-unused-vars
  105:15  warning  Identifier 'td3_1' is not in camel case      camelcase
  105:88  warning  Missing semicolon                            semi
  106:15  warning  'td3_2' is assigned a value but never used   no-unused-vars
  106:15  warning  Identifier 'td3_2' is not in camel case      camelcase
  106:79  warning  Missing semicolon                            semi
  110:49  warning  Missing semicolon                            semi
  111:15  warning  'td4_1' is assigned a value but never used   no-unused-vars
  111:15  warning  Identifier 'td4_1' is not in camel case      camelcase
  111:94  warning  Missing semicolon                            semi
  112:15  warning  'td4_2' is assigned a value but never used   no-unused-vars
  112:15  warning  Identifier 'td4_2' is not in camel case      camelcase
  112:68  warning  Missing semicolon                            semi
  118:76  warning  Missing semicolon                            semi
  121:65  warning  Missing semicolon                            semi
  122:82  warning  Missing semicolon                            semi
  123:93  warning  Missing semicolon                            semi
  124:1   warning  More than 2 blank lines not allowed          no-multiple-empty-lines

✖ 51 problems (0 errors, 51 warnings)
```